presto-main/src/main/java/com/facebook/presto/sql/gen/JoinFilterFunctionCompiler.java

```diff
@@ -71,6 +72,7 @@
 import static com.facebook.presto.sql.gen.TryCodeGenerator.defineTryMethod;
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static com.google.common.base.Verify.verify;
+import static com.google.common.collect.ImmutableList.toImmutableList;
```

Comment:
`JoinFilterCacheKey` currently has `sortChannel` field? 
It seems unnecessary to me. Anyway, it should either be removed. Or taken into consideration in `equals` and `hashCode`.

It is not related strictly to this PR but maybe you could fix that that along the way as you are working on this class anyway. As a separate commit of course :).